src/Acuminator/Acuminator.Analyzers/DiagnosticsShortName.resx

```diff
@@ -0,0 +1,324 @@
+  </data>
+  <data name="PX1048" xml:space="preserve">
+    <value>RowChangesInEventHandlersAllowedForArgsOnly</value>
+  </data>
+  <data name="PX1047" xml:space="preserve">
+    <value>RowChangesInEventHandlersForbiddenForArgs</value>
+  </data>
+  <data name="PX1049" xml:space="preserve">
+    <value>DatabaseQueriesInRowSelected</value>
+  </data>
+  <data name="PX1050" xml:space="preserve">
+    <value>HardcodedStringInLocalizationMethod</value>
+  </data>
+  <data name="PX1051" xml:space="preserve">
+    <value>NonLocalizableString</value>
+  </data>
+  <data name="PX1052" xml:space="preserve">
+    <value>IncorrectStringToFormat</value>
+  </data>
+  <data name="PX1053" xml:space="preserve">
+    <value>ConcatenationPriorLocalization</value>
+  </data>
+  <data name="PX1054" xml:space="preserve">
+    <value>PXGraphLongRunOperationDuringInitialization</value>
+  </data>
+  <data name="PX1057" xml:space="preserve">
+    <value>PXGraphCreationDuringInitialization</value>
+  </data>
+  <data name="PX1058" xml:space="preserve">
+    <value>PXGraphSavingChangesDuringInitialization</value>
+  </data>
+  <data name="PX1055" xml:space="preserve">
+    <value>DacKeyFieldsWithIdentityKeyField</value>
+  </data>
+  <data name="PX1059" xml:space="preserve">
+    <value>ChangesInPXCacheDuringPXGraphInitialization</value>
+  </data>
+  <data name="PX1060" xml:space="preserve">
+    <value>LegacyBqlField</value>
+  </data>
+  <data name="PX1061" xml:space="preserve">
+    <value>LegacyBqlConstant</value>
+  </data>
+  <data name="PX1070" xml:space="preserve">
+    <value>UiPresentationLogicInEventHandlers</value>
+  </data>
+  <data name="PX1071" xml:space="preserve">
+    <value>PXActionExecutionInEventHandlers</value>
+  </data>
+  <data name="PX1072" xml:space="preserve">
+    <value>PXGraphCreationForBqlQueries</value>
+  </data>
+  <data name="PX1073" xml:space="preserve">
+    <value>ThrowingExceptionsInRowPersisted</value>
+  </data>
+  <data name="PX1074" xml:space="preserve">
+    <value>ThrowingSetupNotEnteredExceptionInEventHandlers</value>
+  </data>
+  <data name="PX1075" xml:space="preserve">
+    <value>RaiseExceptionHandlingInEventHandlers</value>
+  </data>
+  <data name="PX1080" xml:space="preserve">
+    <value>DataViewDelegateLongOperationStart</value>
+  </data>
+  <data name="PX1081" xml:space="preserve">
+    <value>PXGraphExecutesActionDuringInitialization</value>
+  </data>
+  <data name="PX1082" xml:space="preserve">
+    <value>ActionExecutionInDataViewDelegate</value>
+  </data>
+  <data name="PX1083" xml:space="preserve">
+    <value>SavingChangesInDataViewDelegate</value>
+  </data>
+  <data name="PX1084" xml:space="preserve">
+    <value>GraphCreationInDataViewDelegate</value>
+  </data>
+  <data name="PX1085" xml:space="preserve">
+    <value>DatabaseQueriesInPXGraphInitialization</value>
+  </data>
+  <data name="PX1086" xml:space="preserve">
+    <value>ThrowingSetupNotEnteredExceptionInLongRunningOperation</value>
```

Comment:
ThrowingSetupNotEnteredExceptionInLongRunOperation
(you have LongRun in PX1054)